tests/end-to-end/external-solution-integration/chart/external-solution/values.yaml

```diff
@@ -3,7 +3,7 @@ containerRegistry:
 
 image:
   dir:
-  version: PR-8511
+  version: PR-8734
```

Comment:
As this image is used in more than one chart and some of the changed code is shared between different e2e scenarios please bump also the versions in:

1. file: `resources/compass-runtime-agent/values.yaml`, value: `global.compassExternalSolutionTests.version`
2. file: `resources/core/values.yaml`, value: `global.e2e_external_solution.version`

to verify if nothing broke.

Reply:
Done. Nice catch 👍